.github/stale.yml

```diff
@@ -22,20 +22,24 @@ exemptMilestones: false
 staleLabel: Review - Abandoned
 
 # Comment to post when marking as stale. Set to `false` to disable
-markComment: >
+markComment: |
```

Comment:
Why change the markComment symbol?

Reply:
I think it's better to keep it folded is better. Accirding to spec, if you want to force a new line in in folded you just need to add a blank line between the test blocks.

Reply:
Fixed.